Azure-Samples/cosmos-db-sql-api-dotnet-samples
# Query doesn't work.

The query specified on line 66 doesn't work.  It should have specified the partition key by name, which is the category, field.

## Purpose
To make the documentation shown on [this page](https://docs.microsoft.com/en-us/azure/cosmos-db/sql/quickstart-dotnet?tabs=azure-cli%2Cwindows) more accurate.  I was following along and found that the query didn't work.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
If you just run the original code, you will see that it does NOT output "Found item" because the query was wrong.  With my change, it will run properly.
```

## What to Check
Verify that the following are valid
* Should get "Found item:     Yamba Surfboard" output to your console app.

## Other Information
